src/tests/ui/planner/iteration.ts

```diff
@@ -58,7 +58,10 @@ export class Iteration extends ui.BaseElement {
     await this.createIterationButton.untilHidden();
   }
   
-  async addNewChildIteration(iterationName: string, parentIteration: string) {
+  async addNewChildIteration(iterationName: string, parentIteration: string, append: boolean = false) {
```

Comment:
Why is this `append` flag needed? 
We should always clear the input and set the parent iteration name.

Reply:
Yes, I'm clearing the input and then setting the parent iteration name